### ISSUE TITLE
Allow bubble to be triggered by keyboard

### ISSUE THREAD

**Issue by Arcanemagus** [steelbrain/linter-ui-default#80]
Currently the UI ignores the keyboard cursor. The tooltip bubble should be triggered by it as well so keyboard only users are not left out.
